oio/event/filters/notify.py

```diff
@@ -32,12 +70,16 @@ def init(self):
 
     def process(self, env, cb):
         data = json.dumps(env)
```

Comment:
We should move l.72 inside the `if`
It may make some differences if there is a lot of account/container to skip